src/basins/edgetracking.jl

```diff
@@ -0,0 +1,194 @@
+export edgetracking, bisect_to_edge
+
+"""
+    edgetracking(ds::DynamicalSystem, u1, u2, attractors::Dict; kwargs...)
+Runs the edge tracking algorithm[^1] for a multistable DynamicalSystem `ds`.
+
+The algorithm is initialized from two states `u1`, `u2` in state space that must belong to
+different basins of attraction. The system's `attractors` are specified as a dictionary of
+`StateSpaceSet`s.
+
+## Keyword arguments
+* `eps1 = 1e-7`: bisection distance threshold
+* `eps2 = 1e-6`: trajectory divergence distance threshold
+* `maxiter = 100`: maximum number of iterations before the algorithm stops
+* `abstol = 0.0`: convergence threshold for returned edge state (distance in state space)
```

Comment:
I don't understand what `abstol` is from this sentence, and you don't explain it more in Description either.

Reply:
I hope I have clarified this better now.